.markdownlint-cli2.shared.cjs

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+
+   config: require(`${__dirname}/.markdownlint.json`),
+
+   globs: [ '**/*.md' ],
+
+   ignores: [
+      "node_modules",
+      "CHANGELOG.md",
```

Comment:
Shouldn't these be single quotes? If so, I have a feeling our linting rules may not be checking `*.cjs` files.